### DISCUSSION TITLE
Is there a reason why FiraCode Nerd Font Mono has no Oblique or Italic?

### DISCUSSION THREAD

**Discussion by unphased** [ryanoasis/nerd-fonts#1136] [Q&A]
For example this has italic. https://github.com/zwaldowski/Fira/tree/zwaldowski/mod-new/otf

**Comment by Finii** [comment]
Well, Fira Code (https://github.com/tonsky/FiraCode) does not have slanted glyphs.
Compare
* https://github.com/tonsky/FiraCode/issues/1266

The font you linked is not `Fira Code` but `Fira`. That would be the `Fira Mono` we have in the `patched-fonts/`. And the font you linked _also_ has no slanted mono variant...:

![grafik](https://user-images.githubusercontent.com/16012374/222357506-f69e1202-77d6-4ebb-af8f-6eff92e64ccb.png)


Of course you can always just take Zwaldowski's `Fira Sans` font and self-patch the symbols in. But that is not a monospaced font.
